mostConsectiveRepeatation.py

```diff
@@ -0,0 +1,38 @@
+# Question 8: Longest Repetition
+
+# Define a procedure, longest_repetition, that takes as input a
+# list, and returns the element in the list that has the most
+# consecutive repetitions. If there are multiple elements that
+# have the same number of longest repetitions, the result should
+# be the one that appears first. If the input list is empty,
+# it should return None.
+
+def longest_repetition(li):
+    # Your code here
+    max = -1
+    count = 0
+    number = None
+    for i in range(len(li)-1):
```

Comment:
`print (longest_repetition([4, 4, 4, 4, 4, 4, 4]))` should display 4 but it says `None`